src/celeritas/io/NeutronXsReader.cc

```diff
@@ -89,7 +89,7 @@ NeutronXsReader::operator()(AtomicNumber atomic_number) const
             // Convert to the celeritas native length 1/[len^2] from
             // clhep::mm^2 as stored in G4PARTICLEXS/neutron/el data
             infile >> result.x[i] >> input_xs.value();
-            result.y[i] = native_value_from(input_xs);
+            result.y[i] = native_value_from(input_xs) / units::barn;
```

Comment:
Since we're going to be using `Barn` in more places, could you add to `celeritas/Quantities.hh`
```c++
using BarnXs = Quantity<Barn>;
```
and then this can be
```suggestion
            result.y[i] = native_value_to<units:: BarnXs>(native_value_from(input_xs)).value();
```
(if that is any clearer...)

Reply:
No a super fan for this^^, but updated as suggested.